src/app/content/studyGuides/components/UsingThisGuide/UsingThisGuideBanner.tsx

```diff
@@ -82,14 +88,32 @@ const BodyWrapper = styled.div`
   display: flex;
   justify-content: center;
 `;
-
 interface Props {
   onClick: () => void;
-  isUTGopen: boolean;
+  show: boolean;
+  isOpenedForTheFirstTime: boolean;
 }
 
 // tslint:disable-next-line:variable-name
 const UsingThisGuideBanner = (props: Props) => {
+  const desktopBannerRef = React.useRef<HTMLImageElement>(null);
+  const toggleCounter = React.useRef(0);
+  const trackOpenUTG = useAnalyticsEvent('openUTG');
+
+  React.useEffect(() => {
+    if (props.show) {
+      toggleCounter.current += 1;
+      Cookies.set(cookieUTG, 'true');
```

Comment:
The cookie needs an expiration date or it will be set as a session cookie.

Reply:
https://openstax.slack.com/archives/C8V1XET1R/p1598632681285700

Reply:
Did you find this in documentation? To me, giving it an expiration date will make it a session cookie, so I'm confused, I don't see expiration dates on the other cookies being used

Reply:
I don't think it's tied to the `js-cookie` but rather default browser behavior.
You can't make a cookie last indefinitely (unless you would refresh it), at best set its expiration date far enough.

> I don't see expiration dates 

That's why [this](https://app.zenhub.com/workspaces/openstax-unified-5b71aabe3815ff014b102258/issues/openstax/unified/1265) was needed. Chrome seems to not be very consistent with clearing session cookies, you can try it out on firefox.